setup.py

```diff
@@ -123,6 +126,48 @@ def get_cuda_path():
     return _cuda_path
 
 
+def get_sycl_path():
+    basekit = os.environ.get("ONEAPI_ROOT")
```

Comment:
Is this the only way to get the path?

Reply:
You need SYCL lib and headers to build XPU extension of mpi4jax. As shown in yml file for testing building of XPU extension we can call  `source $HOME/basekit/setvars.sh`
which is the way to enable Intel oneAPI basekit. By the default it will enable all installed components like Intel components, sycl, MPI etc.  And this process will also set ONEAPI_ROOT var. In theory you could export manually ONEAPI_ROOT env var without calling `setvars.sh` as an alternative solution.

Reply:
For cuda, just a few lines above, if `CUDA_ROOT` is not set we test if `/usr/local/cuda` exists and/or if `nvcc` can be used to get the root.

Maybe it could be possible to try some standard default path and check if it installed there?

Reply:
We discussed this internally and ```ONEAPI_ROOT``` is having an default location : ```/opt/intel/oneapi``` , but inside of this there maybe a couple of compilers(this is where SYCL is). Used compiler  is pointed by  ```CMPLR_ROOT``` env var so we will update PR with using this env var instead of ```ONEAPI_ROOT```.

In case of fallback w.g. no ```CMPLR_ROOT``` is set then we could fallback to most recent SYCL compiler from default location.